docs/docs/overview/quickstart/get-started-with-java.md

```diff
@@ -0,0 +1,342 @@
+---
+sidebar_position: 12
+sidebar_label: Get started with Java
+---
+
+# Java Quickstart
+
+Learn how to integrate Novu into your Java project on the fly and send notifications across different channels (SMS, Email, Chat, Push).
+
+In this Quickstart, you will learn how to:
+
+- [Install and Set up the Novu Java SDK.](#install-and-set-up-the-novu-java-sdk-in-your-project)
+- [Configure a Notification Channel Provider.](#set-up-a-channel-provider)
+- [Set up a Notification Workflow.](#create-a-notification-workflow)
+- [Create a Subscriber and update a Subscriber's information.](#create-a-subscriber-and-update-a-subscribers-information)
+- [Send your first Notification to a Subscriber.](#trigger-a-notification)
+- [Send Notifications via Topics.](#topics)
+
+Let's get started! :muscle:
+
+## Requirements
+
+To be able to use this Quickstart, you would need to have the following:
+
+- A Novu account, if you don't have one yet, [sign up](https://web.novu.co) for free.
+- A working Java development environment with JDK 17 and Springboot Framework of 3.1.2+.
+
+The completed code for this quick start is available on GitHub. Check it out [here](https://github.com/novuhq/novu-java-quickstart).
+
+## Install and Set up the Novu Java SDK in your project
+
+First, you need to import the library into your application.
+If you use Maven, add the dependency to the `pom.xml` file;
+
+```xml
+<dependencies>
+  ...
+  <dependency>
+    <groupId>co.novu</groupId>
+    <artifactId>novu-java</artifactId>
+    <version>1.0.0</version>
+  </dependency>
+</dependencies>
+```
+
+If you use Gradle, add the dependency to the `build.gradle` file;
+
+```groovy
+dependencies {
+  implementation 'co.novu:novu-java:1.0.0'
```

Comment:
There is a new release, pls increase the version to 1.1.0

Reply:
Alright